Libraries/Components/Touchable/TouchableHighlight.js

```diff
@@ -225,7 +275,7 @@ var TouchableHighlight = React.createClass({
 
   render: function() {
     return (
-      <View
+      <TVView
```

Comment:
I'm not convinced of this change. Ideally if you're not targetting tvOS, you also shouldn't be using any tvOS specific components.

How about:
1) is there a way for us to have touchable receive defaultProps on the native side whenever you're running under tvOS
2) provide a TVTouchable that creates a Touchable with a custom viewClass (TVView) and a set of extraProps?